iothub_client/src/blob.c

```diff
@@ -280,6 +284,7 @@ BLOB_RESULT Blob_UploadMultipleBlocksFromSasUri(const char* SASURI, IOTHUB_CLIEN
                                                 /*Codes_SRS_BLOB_02_026: [ Otherwise, if HTTP response code is >=300 then Blob_UploadMultipleBlocksFromSasUri shall succeed and return BLOB_OK. ]*/
                                                 if (result != BLOB_OK || *httpStatus >= 300)
                                                 {
+                                                    LogError("unable to Blob_UploadBlock. Returned value=%d, httpStatus=%u");
```

Comment:
This is missing the parameters

Reply:
Oh.... sorry.... I probably shouldn't code before coffee...